Docs/Components-pages/Alerts-page.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/components/Alerts/Alerts.css">
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="/Docs/Docs.css">
+    <title> Blossom UI/Docs</title>
+</head>
+
+<body>
+
+    <nav>
+        <div>
+            <a href="/index.html" class="main-heading"> Blossom UI </a>
+        </div>
+        <ul class="navigation">
+            <li class="nav-items"> <a class="nav-items" href="/"> Documentaton </a></li>
+            <li class="nav-items"> <a class="nav-items" href="/"> Get Started </a></li>
+        </ul>
+    </nav>
+    <main class="main-section-components-demo">
+        <aside class="side-navbar">
+            <div class="section-heading">
+                Get Started
+            </div>
+            <ul class="list">
+                <li class="component"><a href="#Intro"> Introduction </a></li>
+            </ul>
+            <div class="section-heading">
+                Components
+            </div>
+            <ul class="list">
+                <li class="component active"><a href="/Docs/Components-pages/Alerts-page.html"> Alerts </a></li>
+                <li class="component"><a href="/components/avatar/avatar.html"> Avatars </a></li>
+                <li class="component"><a href="/components/Badges/Badges.html"> Badges </a></li>
+                <li class="component"><a href="/components/Buttons/Buttons.html"> Buttons </a></li>
+                <li class="component"><a href="/components/Cards/cards.html"> Cards </a></li>
+                <li class="component"><a href="/components/Grids/Grids.html"> Grids </a></li>
+                <li class="component"><a href="/components/Inputs/input.html"> Inputs </a></li>
+                <li class="component"><a href="/components/Navigation/Navigation.html"> Navigation </a></li>
+                <li class="component"><a href="/components/responsive-imgs/responsive-imgs.html"> Responsive Images</a></li>
+                <li class="component"><a href="/components/text-utilities/text-utilities.html"> Typography </a></li>
+            </ul>
+        </aside>
+        <section class="content-section">
+            <section class="content-subsection flex">
+                <p class="heading"> Alerts </p>
+                <p class="para-heading">   </p>
```

Comment:
empty `p` tag, could be removed if not necessary

Reply:
Thanks for pointing, will remove that.